armi/reactor/flags.py

```diff
@@ -290,6 +290,17 @@ def fromString(cls, typeSpec):
         .. impl:: Retrieve flag from a string.
             :id: I_ARMI_FLAG_TO_STR0
             :implements: R_ARMI_FLAG_TO_STR
+
+            For a string passed as ``typeSpec``, first converts the whole string
+            to uppercase.
+            Then tries to parse the string for any special phrases, as defined
+            in the module dictionary ``_CONVERSIONS``, and converts those phrases
+            to flags directly.
```

Comment:
```suggestion
            For a string passed as ``typeSpec``, first converts the whole string
            to uppercase. Then tries to parse the string for any special phrases, as
            defined in the module dictionary ``_CONVERSIONS``, and converts those
            phrases to flags directly.
```

Reply:
One line of text only had two words. It was just a bit awkward, so I... justified the text a bit.

Reply:
Right, I miss this part.